Resources/Prototypes/DeltaV/Entities/Structures/Storage/Closets/Lockers/lockers.yml

```
@@ -0,0 +1,25 @@
+- type: entity
+  id: LockerChiefJustice
+  parent: LockerBaseSecure
+  name: chief justice's locker
+  components:
+  - type: Appearance
+  - type: EntityStorageVisuals
+    stateBaseClosed: cj 
+    stateDoorOpen: cj_open
+    stateDoorClosed: cj_door
+  - type: AccessReader
+    access: [["ChiefJustice"]]
+    
+- type: entity
+  id: LockerClerk
+  parent: LockerBaseSecure
+  name: clerk's locker
+  components:
+  - type: Appearance
+  - type: EntityStorageVisuals
+    stateBaseClosed: clerk
+    stateDoorOpen: clerk_open
+    stateDoorClosed: clerk_door
+  - type: AccessReader
+    access: [["Justice"]]
```

```suggestion
    access: [["Justice"]]

```
No prosecutor locker? I'm assuming that's coming soon